components/property.vue

```diff
@@ -1,63 +1,152 @@
 <template>
-  <div class="bg-sidebar sidebar">
+  <div class="sidebar scroll-sect" :class="{ 'scroll-sect-dark': $vuetify.theme.dark, 'scroll-sect-light': !$vuetify.theme.dark }">
     <p class="title">Property</p>
-    <p>{{ selectedLayer }}</p>
     <!-- color controller -->
     <div class="property">
-      <p class="property-title">color</p>
+      <p class="property-title mb-2">Color</p>
       <div class="text-left">
-        <v-menu 
+        <v-menu
           offset-y 
           :close-on-content-click="false"
           >
           <template v-slot:activator="{ on }">
             <v-btn
-              :color="color"
+              :color="selectedLayer.color.hex"
               dark
               v-on="on"
               class="mr-2"
             >
             </v-btn>
-            <span>{{color.slice(0, 7)}}</span>
+            <span>{{ selectedLayer.color.hex }}</span>
           </template>
           <v-color-picker
-            value="#7417BE"
-            v-model="color"
+            light
+            v-model="selectedLayer.color"
             show-swatches
             class="mx-auto"
         ></v-color-picker>
         </v-menu>
       </div>
     </div>
-    
+
+    <!-- opacity controller -->
+    <div class="property mt-7">
+      <div class="d-flex align-items-center mb-2">
+        <p class="property-title m-0">Opacity</p>
+        <v-tooltip bottom nudge-right="65" nudge-bottom="60">
+          <template v-slot:activator="{ on, attrs }">
+            <i
+              class="far fa-question-circle fa-sm ml-2"
+              v-bind="attrs"
+              v-on="on"
+            >
```

Comment:
Is this normal coding convention?
We could follow https://www.w3schools.com/html/html5_syntax.asp

Reply:
For coding convention, are you referring to the multiple lines in a html tag?

Reply:
I have found the multiple lines from the style guide https://www.w3schools.com/html/html5_syntax.asp
Thank you.